src/postgres/src/backend/utils/misc/pg_yb_utils.c

```diff
@@ -1474,3 +1476,99 @@ yb_hash_code(PG_FUNCTION_ARGS)
 	uint16_t hashed_val = YBCCompoundHash(arg_buf, total_bytes);
 	PG_RETURN_UINT16(hashed_val);
 }
+
+/*---------------------------------------------------------------------------*/
+/* Deterministic DETAIL order                                                */
+/*---------------------------------------------------------------------------*/
+
+static int yb_detail_sort_comparator(const void *a, const void *b)
+{
+	return strcmp(*(const char **) a, *(const char **) b);
+}
+
+typedef struct {
+	char **lines;
+	int length;
+} DetailSorter;
+
+void detailSorterFromList(DetailSorter *v, List *litems, int capacity)
```

Comment:
I think the naming convention is that these helper functions should be in underscore case (`detail_sorter_from_list`)

Reply:
Hah, I was confused by this. `List` functions use that type of casing while the `StringInfo` uses the other. Addressed in this commit: https://github.com/yugabyte/yugabyte-db/pull/9650/commits/96eeb6b8a7253ae54952896fb54cb41ea21db32d.